ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -133,10 +133,43 @@ class SignalSpec: QuickSpec {
 		}
 
 		describe("takeWhile") {
-			pending("should take while the predicate is true") {
+			var signal: Signal<Int, NoError>!
+			var observer: Signal<Int, NoError>.Observer!
+
+			beforeEach {
+				let (baseSignal, sink) = Signal<Int, NoError>.pipe()
+				signal = baseSignal |> takeWhile { $0 <= 4 }
+				observer = sink
+			}
+
+			it("should take while the predicate is true") {
+				var latestValue: Int!
+				var completed = false
+
+				signal.observe(next: { value in
+					latestValue = value
+				}, completed: {
+					completed = true
+				})
+
+				sendNext(observer, 4)
+				expect(latestValue).to(equal(4))
+				expect(completed).to(beFalse())
```

Comment:
Fails, or passes? Just not sure since this comment is on the section for a value that passes, and I also don't know what other kind of failing value you tested here,